aten/src/ATen/native/NNPACK.cpp

```diff
@@ -87,7 +87,29 @@ static bool init_nnpack() {
 }
```

Comment:
you also need to add <ATen/Parallel.h> include, it was missing before, and without it INTRA_OP_PARALLEL is undefined.

Reply:
Good point.